cfme/tests/automate/test_instance.py

```diff
@@ -113,3 +113,109 @@ def test_automate_instance_missing(domain, klass, namespace, appliance):
     )
     assert appliance.ssh_client.run_command(
         'grep {} /var/www/miq/vmdb/log/automation.log'.format(catch_string)).success
+
+
+@pytest.mark.tier(1)
+def test_automate_relationship_trailing_spaces(request, klass, namespace, domain):
+    """
+    Handle trailing whitespaces in automate instance relationships.
+
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/10h
+        caseimportance: medium
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.6
+        casecomponent: Automate
+        tags: automate
+        title: Test automate relationship trailing spaces
+        testSteps:
+            1. Create a class and its instance, also create second one,
+               that has a relationship field.
+            2. Create an instance with the relationship field pointing to the first class'
+               instance but add a couple of whitespaces after it.
+            3. Execute the AE model, eg. using Simulate.
+        expectedResults:
+            1.
+            2.
+            3. Logs contain no resolution errors.
+
+    PR:
```

Comment:
No need to mention PR . mention bugzilla instead if you have.

Reply:
There is no BZ for this fix. I found this PR only. So keeping as it is.